internal/pkg/manifest/workload.go

```diff
@@ -24,6 +24,14 @@ const (
 	defaultDockerfileName = "Dockerfile"
 )
 
+// SQS Queue configs
+const (
+	sqsFifoPerMessageGroupID = "perMessageGroupId"
+	sqsFifoPerQueue          = "perQueue"
+	sqsFifoMessageGroup      = "messageGroup"
+	sqsFifoQueue             = "queue"
+)
```

Comment:
I find it helpful to name "enum" field consts after what they represent, the [AWS sdk for example](https://docs.aws.amazon.com/sdk-for-go/api/service/cloudformation/#pkg-constants) does similarly

```suggestion
// SQS Queue field options.
const (
	sqsFIFOThroughputLimitPerMessageGroupID = "perMessageGroupId"
	sqsFIFOThroughputLimitPerQueue          = "perQueue"
	sqsDeduplicationScopeMessageGroup      = "messageGroup"
	sqsDeduplicationScopeQueue             = "queue"
)
```